tests/acceptance/features/apiSpacesShares/shareOperations.feature

```diff
@@ -19,7 +19,7 @@ Feature: sharing
     When user "Brian" gets the following properties of file "/tmp.txt" inside space "Shares" using the WebDAV API
       | propertyName          |
       | ocs:share-permissions |
-    Then the HTTP status code should be "200"
+    Then the HTTP status code should be "207"
```

Comment:
will this scenario pass or fail?

Reply:
yes while getting the properties, listfile file api is called which gives multistatus status code 207.
Previously this response was not set and the response from the given step was used. So these then steps checked for status 200 and 201 in the previous feature files set by the previous given step but not when step.

Reply:
So, now we are doing it right